{"name": "Проверка ошибки при создании двух одинаковых курьеров", "status": "passed", "description": "Отправляем повторный запрос на создание курьера, проверяем ответ и удаляем курьера", "start": 1719311644034, "stop": 1719311644312, "uuid": "c21f635b-d1e5-4ab2-84d2-9b39ab6a607e", "historyId": "cf269a25aa8f451cb51d06fe26b80fd6", "testCaseId": "cf269a25aa8f451cb51d06fe26b80fd6", "fullName": "tests.test_create_courier.TestCreateCourier#test_registration_double_courier_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "WIN-DIQJIRLC2O2"}, {"name": "thread", "value": "23404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}